Makefile

```diff
@@ -49,6 +44,10 @@ dev: check-docker
 bootstrap: check-docker
 	${DEV_ENV_CMD} glide install
 
+# This is so you can build the binary without using docker
+build-binary:
+	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -a -installsuffix cgo -ldflags ${LDFLAGS} -o $(BINARY_DEST_DIR)/logger main.go
```

Comment:
I'd rather have this build inside the `go-dev` docker container, as we do in other projects